aws/table_aws_bedrock_foundation_model.go

```diff
@@ -0,0 +1,151 @@
+		},
+		Get: &plugin.GetConfig{
+			KeyColumns: plugin.SingleColumn("model_id"),
+			Hydrate:    getBedrockFoundationModel,
+			Tags:       map[string]string{"service": "bedrock", "action": "GetFoundationModel"},
+		},
+		GetMatrixItemFunc: SupportedRegionMatrix(AWS_BEDROCK_SERVICE_ID),
+		Columns: awsRegionalColumns([]*plugin.Column{
+			{
+				Name:        "arn",
+				Description: "The Amazon Resource Name (ARN) of the foundation model.",
+				Transform:   transform.FromField("ModelArn"),
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "model_id",
+				Description: "The unique identifier of the foundation model.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "model_name",
+				Description: "The name of the foundation model.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "provider_name",
+				Description: "The name of the model provider.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "input_modalities",
+				Description: "The input modalities supported by the model.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "output_modalities",
+				Description: "The output modalities supported by the model.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "customizations_supported",
+				Description: "The customizations supported by the model.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "inference_types_supported",
+				Description: "The inference types supported by the model.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "model_lifecycle",
+				Description: "The lifecycle status of the model.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "response_streaming_supported",
+				Description: "Whether the model supports response streaming.",
+				Type:        proto.ColumnType_BOOL,
+			},
+			// Steampipe standard columns
+			{
+				Name:        "title",
+				Description: "Title of the resource.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("ModelName"),
+			},
+			{
+				Name:        "akas",
+				Description: "Array of globally unique identifier strings (also known as) for the resource.",
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromField("ModelArn").Transform(transform.EnsureStringArray),
+			},
+		}),
+	}
+}
+
+func listBedrockFoundationModels(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	// Create AWS service client
+	svc, err := BedrockClient(ctx, d)
+
+	if svc == nil {
+		return nil, nil
+	}
+
+	if err != nil {
+		plugin.Logger(ctx).Error("aws_bedrock_foundation_model.listBedrockFoundationModels", "connection_error", err)
+		return nil, err
+	}
+
+	// Execute list call
+	resp, err := svc.ListFoundationModels(ctx, &bedrock.ListFoundationModelsInput{})
```

Comment:
Could you please add a comment noting that the API does not currently support pagination, so we can track and revisit it in the future?